services/cms.ts

```diff
@@ -0,0 +1,53 @@
+import { ContentfulClientApi, EntryCollection, createClient } from "contentful";
+import {
+  IFAQItem,
+  IFetchEntriesReturn,
+  IFetchFAQItemsReturn,
+} from "@/types/cms";
```

Comment:
This import won't work because the `tsconfig.json` file hasn't been updated to support using `@`. See this https://www.typescriptlang.org/tsconfig#paths and then the session website `tsconfig.json`

Reply:
Also we need to copy the type files from session-website. They don't exist in this repository yet.